async_packager/setup.cfg

```diff
@@ -0,0 +1,39 @@
+[metadata]
+name = cumulus-packager
+version = 0.1.0
+author = USACE RSGIS
+author_email = author@example.com
+description = Packager Implementation
+long_description = file: README.md, LICENSE
+license = MIT
+long_description_content_type = text/markdown
+url = https://docs.rsgis.dev
```

Comment:
This url looks bogus.  not sure it really matters at this point.

Reply:
More of a placeholder.  Should have a better url.